blog/thirdweb.md

```diff
@@ -0,0 +1,260 @@
+---
+slug: thirdweb-wallet-upload
+hide_table_of_contents: false
+title: "Introducing Wallet SDK and Sending remote storage"
+date: 2023-05-17
+authors: fabri
+image: "https://blog.thirdweb.com/content/images/size/w2000/2023/05/How-to-use-smart-wallet-.png"
+description: "Sending remote attachments with XMTP"
+tags:
+- Content Types
+- SDKs
+- Media
+- Developers
+- WalletSDK
+- Thirdweb
+---
+
+Creating an effective 'Connect Wallet' flow is a challenging aspect of Web3 development. Thankfully, the team at Thirdweb has developed an excellent SDK, simplifying this process and enabling a superior wallet experience for your users.
+
+<!--do we want to request a blog card visual from Nad?-->
+
+<!--truncate-->
+
+## Concepts
+
+### Thirdweb WalletSDK
+The WalletSDK is a development kit that grants developers access to a comprehensive selection of wallets, ranging from custodial to MPC to smart contracts.
+[Read more](https://twitter.com/thirdweb/status/1654191962751389697)
+
+### XMTP content types
```

Comment:
We follow the Google developer doc style guide for headings - so sentence case and imperative mood for tasks: https://developers.google.com/style/headings. I've updated the heading according to this style guidance. =)